app/templates/src/main/webapp/scss/bootstrap.scss

```diff
@@ -0,0 +1,5 @@
+$icon-font-path: "../../bower_components/bootstrap-sass/assets/fonts/bootstrap/"; 
+ 
```

Comment:
Add a comment here saying that actual bootstrap css will be generated from the bower bootstrap-sass